### ISSUE TITLE
no-descending-specificity with "empty" definition

### ISSUE THREAD

**Issue by bkdotcom** [stylelint/stylelint#4636]
I apologize in advance for my inability to describe this issue...

I have this snippet

```css
    nav {
      display: inline-block;
      a {
        display: inline-block;
      }
    }

    a,
    button,
    label {
      &:focus,
      &:hover {
        color: #333;
        text-decoration: none;
      }
    }
```

results in 
`Expected selector "a" to come before selector "nav a"`
(there is no "a" selector... only a:focus & a:hover)!

swapping to 
```css
    a,
    button,
    label {
      &:focus,
      &:hover {
        color: #333;
        text-decoration: none;
      }
    }

    nav {
      display: inline-block;
      a {
        display: inline-block;
      }
    }
```
results in 
`Expected selector "nav a" to come before selector "a:focus"`

feels like a bug..   as no `a` selector is generated
if not a bug, how should this be written?
